newshour_transcript_cleanup.py

```diff
@@ -1,39 +1,35 @@
-import json
-import pathlib
+import os
 import re
+import argparse
+
+def extract_text(original_transcript_path, cleaned_transcript_path):
```

Comment:
also let's make it take just a string (read from a file) as the input, refactoring file-handling processes to outside functions.